kube-runtime/src/watcher.rs

```diff
@@ -217,6 +233,18 @@ pub struct Config {
     /// Configures re-list for performance vs. consistency.
     pub list_semantic: ListSemantic,
 
+    /// Kubernetes 1.27 Streaming Lists
+    /// Mode for the initial watch call, or when the watcher is restarted.
+    /// Enables opening a stream to bring data from the API server.
+    /// Streaming has the primary advantage of using fewer server's resources to fetch data.
+    ///
+    /// The old behaviour establishes a LIST request which gets data in chunks.
+    /// Paginated list is less efficient and depending on the actual size of objects
+    /// might result in an increased memory consumption of the APIServer.
```

Comment:
This comment seems a bit wrong, it's written as if users have already chosen streaming lists, when that's an option in the enum.